psi4/driver/procrouting/dft_funcs/dict_dh_funcs.py

```diff
@@ -0,0 +1,142 @@
+    "description": '    B2PLYP Double Hybrid Exchange-Correlation Functional\n',
+}
+
+dsd_blyp = {
+    "name": "DSD-BLYP",
+    "x_functionals": {"GGA_X_B88": {"alpha": 0.30}, "X_HF": {"alpha": 0.70}},
+    "c_functionals": {"GGA_C_LYP": {"alpha": 0.56}, "C_MP2": {"alpha": 1.0, "os": 0.46, "ss": 0.40}},
+    "citation": '    S. Kozuch, Phys. Chem. Chem. Phys., 13, 20104, 2011\n',
+    "description": '    DSD-BLYP Dispersion-corrected SCS Double Hybrid XC Functional\n',
+}
+
+core_dsd_blyp = {
+    "name": "CORE-DSD-BLYP",
+    "x_functionals": {"GGA_X_B88": {"alpha": 0.31}, "X_HF": {"alpha": 0.69}},
+    "c_functionals": {"GGA_C_LYP": {"alpha": 0.54}, "C_MP2": {"alpha": 1.0, "os": 0.46, "ss": 0.37}},
+    "citation": '    S. Kozuch, Phys. Chem. Chem. Phys., 13, 20104, 2011\n',
+    "description": '    DSD-BLYP Dispersion-corrected SCS Double Hybrid XC Functional\n' + \
+                   '    (full-core parameterization) \n'  
+}
+
+pbe0_2 = {
+    "name": "PBE0-2",
+    "x_functionals": {"GGA_X_PBE": {"alpha": 0.206299}, "X_HF": {"alpha": 0.793701}},
+    "c_functionals": {"GGA_C_PBE": {"alpha": 0.5}, "C_MP2": {"alpha": 0.5}},
+    "citation": '    J. Chai, Chem. Phys. Lett., 538, 121-125, 2012\n',
+    "description": '    PBE0-2 Double Hybrid Exchange-Correlation Functional\n',
+}
+
+pbe0_dh = {
+    "name": "PBE0-DH",
+    "x_functionals": {"GGA_X_PBE": {"alpha": 0.5}, "X_HF": {"alpha": 0.5}},
+    "c_functionals": {"GGA_C_PBE": {"alpha": 0.875}, "C_MP2": {"alpha": 0.125}},
+    "citation": '    E. Bremond, C. Adamo, J. Chem. Phys., 135, 024106, 2011\n',
+    "description": '    PBE0-DH Double Hybrid Exchange-Correlation Functional\n',
+}
+
+dsd_pbep86 = {
+    "name": "DSD-PBEP86",
+    "x_functionals": {"GGA_X_PBE": {"alpha": 0.32}, "X_HF": {"alpha": 0.68}},
+    "c_functionals": {"GGA_C_P86": {"alpha": 0.45}, "C_MP2": {"alpha": 1.0, "os": 0.51, "ss": 0.23}},
+    "citation": '    S. Kozuch, Phys. Chem. Chem. Phys., 13, 20104, 2011\n',
+    "description": '    DSD-PBEP86 Dispersion-corrected SCS Double Hybrid XC Functional (opt. for -D2)\n',
+}
+
+dsd_pbepbe = {
+    "name": "DSD-PBEPBE",
+    "x_functionals": {"GGA_X_PBE": {"alpha": 0.34}, "X_HF": {"alpha": 0.66}},
+    "c_functionals": {"GGA_C_PBE": {"alpha": 0.51}, "C_MP2": {"alpha": 1.0, "os": 0.53, "ss": 0.12}},
+    "citation": '    S. Kozuch, Phys. Chem. Chem. Phys., 13, 20104, 2011\n',
+    "description": '    DSD-PBEPBE Dispersion-corrected SCS Double Hybrid XC Functional\n',
+}
+
+b2gpplyp = {
+    "name": "B2GPPLYP",
+    "x_functionals": {"GGA_X_B88": {"alpha": 0.35}, "X_HF": {"alpha": 0.65}},
+    "c_functionals": {"GGA_C_LYP": {"alpha": 0.64}, "C_MP2": {"alpha": 0.36}},
+    "citation": '    A. Karton, A. Tarnopolsky, J.-F. Lamere, G. C. Schatz, J.M. L. Martin, J.Phys. Chem. A, 112, 12868-12886,2008   \n',
+    "description": '    B2GPPLYP Double Hybrid Exchange-Correlation Functional\n',
+}
+
+beta = 5.0 * (36.0 * 3.141592653589793)**(-5.0 / 3.0)
```

Comment:
We need to do something to namespace this bit. As a first thought, can you build a function that looks like:

```python
def pwpb95_pw91_tweaks():
    ....
   return [a_pw6, b_pw6, c_pw6, d_pw6, f_pw6, alpha_pw6, expo_pw6]

 {"GGA_X_PW91": {"tweak": pwpb95_pw91_tweaks() ,"alpha": 0.50}
```